packages/common/test-utils.js

```diff
@@ -107,3 +109,83 @@ async function createQueue() {
   return url.format(returnedQueueUrl);
 }
 exports.createQueue = createQueue;
+
+/**
+ * Read a file and return a promise with the data
+ *
+ * Takes the same parameters as fs.readFile:
+ *
+ * https://nodejs.org/docs/v6.10.3/api/fs.html#fs_fs_readfile_file_options_callback
+ *
+ * @param {string|Buffer|integer} file - filename or file descriptor
+ * @param {any} options - encoding and flag options
+ * @returns {Promise} - the contents of the file
+ */
+function promisedReadFile(file, options) {
+  return new Promise((resolve, reject) => {
+    readFile(file, options, (err, data) => {
+      if (err) reject(err);
+      else resolve(data);
+    });
+  });
+}
+
+/**
+ * Validate an object using json-schema
+ *
+ * Issues a test failure if there were validation errors
+ *
+ * @param {Object} t - an ava test
+ * @param {string} schemaFilename - the filename of the schema
+ * @param {Object} data - the object to be validated
+ * @returns {boolean} - whether the object is valid or not
+ */
+async function validateJSON(t, schemaFilename, data) {
+  const schema = await promisedReadFile(schemaFilename, 'utf8').then(JSON.parse);
```

Comment:
What happens if the schema file does not exist?

Reply:
The assumption is that you would only add this check to your test if the schema file exists.

Reply:
It might be nice to just comment the expected behavior or spit out an error in that case. I don't love that the schema locations are assumed, but think it's fine for now and we can make updates if we run into a different scenario.